### DISCUSSION TITLE
Where are the colours calculated differently on iOS and Android?

### DISCUSSION THREAD

**Discussion by pke** [osamaqarem/react-native-image-colors#67] [General]
I am wondering why not both systems get the same colours?

**Comment by osamaqarem** [comment]
Android Palette and iOS's UIImageColors have different internal implementations. This library exposes them to React Native.

README:
> This module is a wrapper around the [Palette](https://developer.android.com/reference/androidx/palette/graphics/Palette) class on Android, [UIImageColors](https://github.com/jathu/UIImageColors) on iOS and [node-vibrant](https://github.com/Vibrant-Colors/node-vibrant) for the web.

**Reply by osamaqarem** [reply]
We could rewrite the package using a shared implementation but that's not something I have time for at the moment. An easy option is to do the calculation at your backend using a different package. This also has the advantage of a small performance improvement for your app.
